src/Smalot/PdfParser/Tests/Units/Page.php

```diff
@@ -107,4 +107,281 @@ public function testGetText()
         $this->assert->string($text)->contains('Courier New');
         $this->assert->string($text)->contains('Verdana');
     }
+    
+    public function testExtractRawData()
+    {
+        // Document with text.
+        $filename = __DIR__ . '/../../../../../samples/Document1_pdfcreator_nocompressed.pdf';
+        $parser   = new \Smalot\PdfParser\Parser();
```

Comment:
I assume you need to re-run PHP-CS-Fixer, because multiple spaces between the variable and `=` are not allowed. I tested it locally with your changes.